src/Microsoft/Provider.php

```diff
@@ -156,6 +156,8 @@ protected function getUserByToken($token)
             $formattedResponse['tenant'] = json_decode((string) $responseTenant->getBody(), true)['value'][0] ?? null;
         }
 
+        $formattedResponse['roles'] = $this->getRoles();
```

Comment:
Can we please get an example in the readme docs of the format of this please

Reply:
@atymic The returned role names' format would depend on how you configure the Entra ID side... but I updated README from what I know....would this work?

Reply:
Gotcha, it returns an array of string for user group names. updated readme and also line 233 accordingly.